firstvoices/backend/views/sites_views.py

```diff
@@ -107,7 +114,7 @@ def list(self, request, *args, **kwargs):
         Return a list of sites grouped by language.
         """
         # retrieve visible sites in order to filter out empty languages
-        sites = Site.objects.visible(self.request.user)
+        sites = Site.objects.filter(visibility__gte=Visibility.MEMBERS)
```

Comment:
Will this change mean that a language admin will not see a team site that they belong to on the explore languages page?

Reply:
Yes, correct. Intentionally

Reply:
It still shows in the "my sites" section at the top of the page